okhttp/src/main/kotlin/okhttp3/CacheControl.kt

```diff
@@ -186,26 +183,26 @@ class CacheControl internal constructor(
      * Sets the maximum age of a cached response. If the cache response's age exceeds [maxAge], it
      * will not be used and a network request will be made.
      *
-     * @param maxAge a non-negative integer. This is stored and transmitted with [TimeUnit.SECONDS]
+     * @param maxAge a non-negative duration. This is stored and transmitted with [TimeUnit.SECONDS]
      *     precision; finer precision will be lost.
      */
-    @ExperimentalOkHttpApi
-    fun maxAge(
-      maxAge: Int,
-      timeUnit: DurationUnit,
-    ) = commonMaxAge(maxAge, timeUnit)
+    @JvmSynthetic // Don't expose @JvmInline types in the Java API.
```

Comment:
You shouldn't need any of these annotations as functions accepting value types should already have their names mangled to include a hypen which is an invalid character in Java method names. The API dump should confirm.

Reply:
Fixed.